### PR TITLE
javaScript file

### PR DESCRIPTION
**Pull request by Frelly0**
This my portfolio page and i added same features:

- script.js
- updated html file and css file

### CODE REVIEW

**Comment by Frelly0** [comment]
Hello Webhint refused to work since last week.

**Comment by Frelly0** [comment]
Hello.. webhint refused to work since last week.

**Review by V-Blaze — CHANGES_REQUESTED** [review]
Hi @Frelly0  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

dcentl8-9b96e129-9725-401b-ac34-2feb990202a0

## Invalid Code Review Request

## You have submitted a project, that doesn't have Javascript `Eslint` installed. According to the requirement for this milestone, you are meant to update your linters to include eslint. I'll suggest you visit the [Microverse linters repository](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for the step guides on how to set up `html-css-js` linters. Also, remember to update your `linters.yml` file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Frelly0** [comment]
Hello. Webhint refused to work since last week.

File: index.html [hunk 1]
```diff
@@ -11,13 +11,20 @@
     <div class="container">
       <nav class="top-menu">
         <div class="sub-menu">
+          
           <h1>My-Frelly</h1>
```
Comment (divinecharlotte) [inline index.html:15]: - [ ] kindly consider sticking to the Figma design as much as possible to make your design interesting and attractive so in order to achieve that I suggest you make sure that this logo is not visible on the mobile menu when it opens and also consider removing the blue background on it when the navbar is not open as well to make your design and exact duplicate of the given one check below how it's looking and fix it
Screen Shot 2023-06-27 at 07 31 17

Screen Shot 2023-06-27 at 07 31 57

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team  , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Clean code 👍 
- Descriptive Readme 👍 

 




 

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Frelly0** [comment]
Hello. Webhint refused to work since last week i don't what to do now.

**Review by LionRouge1 — APPROVED** [review]
Hi **Team**, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**